tests/testthat/test-cube-pairwise.R

```diff
@@ -0,0 +1,166 @@
+                             0.773583582093158, 0.999999999998836, 0.192375246184738, 0.99999993076784, 
+                             0.999991395396033, 1, 0.017277623171216, 3.29012189337341e-06, 
+                             0.99999999999994, 0.999998237045896, 0.0365273119329589, 0.999999857555538, 
+                             2.23456306602809e-06, 0.435830186989942, 0.806407150042716, 0.017277623171216, 
+                             1, 0.999999977981595, 0.821586701043061, 0.982573114952466, 0.999999169027016, 
+                             0.998041030837588, 0.999934687968906, 0.171365670494448, 0.380296648898666, 
+                             3.29012189337341e-06, 0.999999977981595, 1, 0.52406354520284, 
+                             0.926322806048378, 0.99999998900118, 0.981100354607917, 0.999999991067971, 
+                             0.999999931581745, 0.999999999961875, 0.99999999999994, 0.821586701043061, 
+                             0.52406354520284, 1, 0.999999992799126, 0.883025655503086, 0.99999999998941, 
+                             0.33149560264078, 0.999999979427862, 0.999996333717649, 0.999998237045896, 
+                             0.982573114952466, 0.926322806048378, 0.999999992799126, 1, 0.997674862917282, 
+                             0.99999999995011, 0.81726901111794, 0.726740806122402, 0.773583582093158, 
+                             0.0365273119329589, 0.999999169027016, 0.99999998900118, 0.883025655503086, 
+                             0.997674862917282, 1, 0.998461227115608, 0.999994436499243, 0.999997338047414, 
+                             0.999999999998836, 0.999999857555538, 0.998041030837588, 0.981100354607917, 
+                             0.99999999998941, 0.99999999995011, 0.998461227115608, 1, 0.925999125959122, 
+                             0.105707739899106, 0.192375246184738, 2.23456306602809e-06, 0.999934687968906, 
+                             0.999999991067971, 0.33149560264078, 0.81726901111794, 0.999994436499243, 
+                             0.925999125959122, 1), .Dim = c(10L, 10L),
+                           .Dimnames = list(c("1", "2", "3", "4", "5", "6", "7", "8", "9", "10"),
+                                            c("1", "2", "3", "4", "5", "6", "7", "8", "9", "10")))
+    out <- wishartPvalues(hirotsu)
+    expect_equal(out, reference)
+    
+    out_t <- wishartPvalues(hirotsuTranspose, dim="row")
+    expect_equal(out, out_t)
+})
+
+test_that("a matrix where each row and column is the same", {
+    referenceChisq <- structure(rep(0, 16),
+                           .Dim = c(4L, 4L),
+                           .Dimnames = list(c("a", "b", "c", "d"), c("a", "b", "c", "d")))
+    referencePvals <- structure(rep(1, 16),
+                                .Dim = c(4L, 4L),
+                                .Dimnames = list(c("a", "b", "c", "d"), c("a", "b", "c", "d")))
```

Comment:
These could be re-written as
`cubify(0, dims = list(c("a", "b", "c", "d"), c("a", "b", "c", "d")))`
and 
`cubify(1, dims = list(c("a", "b", "c", "d"), c("a", "b", "c", "d")))`
respectively (and most or all other `structure(...)` calls could be as well)

Reply:
*should* they? is `cubify` preferred for developer readability? As only an occasional contributor I wasn’t clear on what `cubify` did and `structure` was clearer to me ”this is the reference object”. Can change if you like.

Reply:
I find cubify to be a bit more readable, and it saves arguments — it's not _necessary_ of course but those structures stood out to me and I had to copy/paste them and run them to confirm they were doing what they were doing.

It's not relevant for these two, but for the ones that are not a single value it also would be nice if the line breaks matched the rows breaks to make it easier to read. I found myself having to mentally find the diagonal and shift forward/back from there to see where I was in the matrix.